packages/vats/test/test-name-hub.js

```diff
@@ -143,3 +143,27 @@ test('makeNameHubKit - default and set', async t => {
     message: 'key "not set" is not already initialized',
   });
 });
+
+test('makeNameHubKit - listen for updates', t => {
```

Comment:
did you want to add a test like https://github.com/Agoric/agoric-sdk/pull/5948 ?

Reply:
yes, but that's not landed yet. hm.

Reply:
Just got approved so it's going in soon. But if you want to follow up after merging this that works for me.